external-apis/muoversi2015.e015.servizirl.it.yaml

```diff
@@ -26,16 +28,41 @@ info:
     ```
     'https://muoversi2015.e015.servizirl.it/planner/rest/soluzioniJson/e015Search?param=milano&maxResult=1'
     ```
+  x-lifecycle:             # {type: string, format: date }
+    published: 2018-01-01  # data di pubblicazione
+    deprecated: 2020-12-31 # data inizio dismissione
+    retired: 2021-03-31    # data termine dismissione
+    maturity: published    # {type: string, enum: [proposal,developing,published,deprecated,retired]
+
+  x-catalog:
+    tag: [orari, trasporto pubblico, arrivo, partenza intermodalità, autobus, tram, treno, traghetto, funicolare, metropolitana]
+    category: [CATEGORY1,CATEGORY2]
+    context:
+      - name: [Nome contesto]
+        description: [Descrizione contesto]
+    ecosystem:
+      - name: E015
+        url: [Url verso ecosistema di cui fa parte questa definizione API]
+        description: [Descrizione Ecosistema]
   contact:
     name: Technical Management Board di E015
     url: http://www.e015.regione.lombardia.it/PE015/esplora-i-contenuti/i-servizi/muoversi-lombardia
     email: e015-tmb@regione.lombardia.it
-  version: 1.0.0
   termsOfService: >-
     http://www.e015.regione.lombardia.it/PE015/system/files_force/allegati/policy-servizio/policy_servizio_muoversi_in_lombardia_0.pdf?download=1&download=1
```

Comment:
Broken link.

Reply:
Forse questo è quello nuovo:
http://www.e015.regione.lombardia.it/site/download-policy?id=13

Reply:
Thx++